src/App.tsx

```diff
@@ -15,36 +17,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+ enum SortType {
+  alphabetically = 'alphabetically',
+  byLength = 'byLength',
+  default = '',
+}
+
+type SortObj = {
+  sortField: SortType;
+  reverseGood?: boolean;
+};
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, reverseGood = false }: SortObj) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.alphabetically:
+          return good1.localeCompare(good2);
+        case SortType.byLength:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverseGood) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
```

Comment:
The function name `getPreparedGoods` does not start with a verb which may make it less clear what the function does. Consider renaming it to something like `prepareGoods` to better reflect the action it performs and adhere to the [CODE STYLE] checklist.